Stop publishing cuspatial packages - v25.06 (#588)
rapidsai/docs: _notices/rsn0045.md

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 45 # should match notice number
notice_pin: true # set to true to pin to notice page

title: "Dropping of Publishing cuSpatial Packages in RAPIDS Release v25.06"
notice_author: RAPIDS TPM
notice_status: In Progress
notice_status_color: yellow
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change

> notice_rapids_version: "v24.10+"

Reviewer: ```suggestion
notice_rapids_version: "v25.06+"
```

notice_created: 2025-04-09
# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2025-04-09
---

> ## Overview
> 
> RAPIDS is will stop publishing cuSpatial packages in RAPIDS Release v25.06, scheduled for June 05, 2025. Development for the cuspatial repository will be paused and no new packages will be published. 
> 
> 
> ## Impact
> 
> Development for the cuspatial repository will be paused and no new packages will be published.

Reviewer: ```suggestion
## Overview

RAPIDS will stop publishing cuSpatial packages in RAPIDS Release v25.06, scheduled for June 05, 2025. Development for the cuspatial repository will be paused and no new packages will be published.

## Impact

Development for the cuspatial repository will be paused and no new packages will be published.
cuSpatial will be dropped from the `rapids` conda metapackage and RAPIDS Docker containers.
```

> ## Overview
> 
> RAPIDS will stop publishing cuSpatial packages in RAPIDS Release v25.06, scheduled for June 05, 2025. Development for the cuspatial repository will be paused and no new packages will be published.
> 
> ## Impact
> 
> Development for the cuspatial repository will be paused and no new packages will be published.
> cuSpatial will be dropped from the `rapids` conda metapackage and RAPIDS Docker containers.

Reviewer: ```suggestion
## Overview

RAPIDS will stop publishing cuSpatial packages in RAPIDS Release v25.06, scheduled for June 05, 2025. Development for the cuSpatial repository will be paused and no new packages will be published.

## Impact

Development for the cuSpatial repository will be paused and no new packages will be published.
cuSpatial will be dropped from the `rapids` conda metapackage and RAPIDS Docker containers.
```
